app/uk/gov/hmrc/tariffclassificationfrontend/controllers/ReassignCaseController.scala

```diff
@@ -47,24 +47,24 @@ class ReassignCaseController @Inject()(authenticatedAction: AuthenticatedAction,
 
   private lazy val form: Form[String] = ReleaseCaseForm.form
 
-  def showAvailableQueues(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
-    getCaseAndRenderView(reference, c =>
+  private def reassignToQueue(f: Form[String], caseRef: String)
```

Comment:
This code was repeated twice. 
I think now the code here is more readable and maintainable.